util/watch/watch.go

```diff
@@ -85,7 +85,13 @@ func batchPublish(container string) {
 		<-time.After(time.Second)
 		if len(changeList) > 0 {
 			lumber.Info("watcher: pushing: %+v", changeList)
-			util.DockerExec(container, "root", "touch", append([]string{"-c"}, changeList...), nil)
+			// util.DockerExec(container, "root", "touch", append([]string{"-c"}, changeList...), nil)
+			touches := []string{}
+			for _, change := range changeList {
+				touches = append(touches, fmt.Sprintf("touch -c -r %s %s", change, change))
+			}
+
+			util.DockerExec(container, "root", "bash", append([]string{"-c"}, strings.Join(touches, " && ")), nil)
```

Comment:
If we try to `touch` a file that has been removed, `-c` will prevent it from being created again, but `-r <deleted_file>` will throw an error message, and `touch` will return `1` instead of `0`. So... We should probably switch from `&&` to `;` here...